### DISCUSSION TITLE
SS total decreased on my osu!mania profile randomly on the stable website

### DISCUSSION THREAD

**Discussion by ItsCollector** [ppy/osu#31180] [Q&A] [answered]
Not sure if it's a bigger problem at hand but I wanted to share this anyway. I had around 160 SSs on my profile last night but I got on this morning and it dropped to just 24. I SS one more map just now to see if it would automatically update to normal but it just increased to 25. 

![image](https://github.com/user-attachments/assets/3b524b82-133e-4539-8d0f-7d830d3d7c2c)

Any thoughts? Here is my [profile](https://osu.ppy.sh/users/21530808)

**Comment by ghost** [comment]
me too

**Comment by felipemarins** [comment]
It seems that there was some issue with the grade recalc. Peppy is already looking into fixing it by re-running the process.

**Comment by peppy** [comment, marked as answer]
This should be fixed now.
